dask/dataframe/groupby.py

```diff
@@ -1984,6 +1984,34 @@ def rolling(self, window, min_periods=None, center=False, win_type=None, axis=0)
             axis=axis,
         )
 
+    def _fillna_groups(self, groups, **kwargs):
```

Comment:
This function doesn't need to be a member function (note we don't use `self` anywhere), so it's probably cleaner and less likely to run into serialization issues to have it be a top-level function

Reply:
That makes sense, I'll update it!